addons/mail/static/src/discuss/discuss.xml

```diff
@@ -10,9 +10,21 @@
             <div class="px-3 bg-100 border-bottom d-flex flex-shrink-0" style="height:42px;">
                 <ThreadIcon className="'me-2 align-self-center'" thread="thread"/>
                 <div class="d-flex flex-grow-1 align-self-center align-items-center">
-                    <div class="lead flex-shrink-0 px-1 border text-truncate fw-bold border-light" t-att-title="thread.name" >
-                        <t t-esc="thread.name"/>
-                    </div>
+                    <t t-if="!state.isEditingThreadName">
+                        <div class="lead flex-shrink-0 px-1 border text-truncate fw-bold" t-att-title="thread.name" t-on-click="onClickThreadName" t-on-mouseenter="onMouseEnterThreadName" t-on-mouseleave="onMouseLeaveThreadName" t-attf-class="{{ state.isMouseOverThreadName ? 'o-threadNameEditable bg-white' : 'border-light' }}">
```

Comment:
`isMouseOverThreadName` can certainly just be some pure classname that does this hover styling.
If not, then can just use `:hover` in css

Reply:
cause there is a check on `isRenameable`, I think it is easier to use a state to handle this rather than using :hover